inc/patches/matrixssl_matrixsslConfig.h.patch

```diff
@@ -12,6 +12,24 @@
  #define USE_TLS_RSA_WITH_AES_256_CBC_SHA256 /* TLS 1.2 */
  /* #define USE_TLS_RSA_WITH_AES_128_GCM_SHA256 */ /* TLS 1.2 */
  /* #define USE_TLS_RSA_WITH_AES_256_GCM_SHA384 */ /* TLS 1.2 */
+@@ -224,14 +224,14 @@ extern "C" {
+ 	SSL_SESSION_ENTRY_LIFE is in milliseconds, minimum 0
+ 
+ */
+-#define SSL_SESSION_TABLE_SIZE	32
+-#define SSL_SESSION_ENTRY_LIFE	86400 * 1000 /* one day */
++/* #define SSL_SESSION_TABLE_SIZE	32 */
++/* #define SSL_SESSION_ENTRY_LIFE 	86400 * 1000 */ /* one day */
```

Comment:
Maybe `ifndef` will be better here, so it won't break if Makefile.PL won't provide these defines?